pkg/vault/test/foundry/BufferVaultPrimitive.t.sol

```diff
@@ -574,6 +574,118 @@ contract BufferVaultPrimitiveTest is BaseVaultTest {
         batchRouter.swapExactIn(paths, MAX_UINT256, false, bytes(""));
     }
 
+    /********************************************************************************
+                          Add/Remove Liquidity from Buffers
+    ********************************************************************************/
+
+    function testAddLiquidityToBuffer() public {
+        BufferAndLPBalances memory beforeBalances = _measureBuffer();
+
+        vm.prank(lp);
+        uint256 lpShares = router.addLiquidityToBuffer(waDAI, _wrapAmount, _wrapAmount, lp);
```

Comment:
Here and on remove, maybe there's a benefit to making these different amounts - i.e., _wrapAmount and _underlyingAmount. Similar to testing weighted pools with 50/50 weights all the time, it's possible something could be backwards and you wouldn't know if the amounts are always equal. A lot of our tests are like that.